src/hooks/useHashConnect/useHashConnect.ts

```diff
@@ -28,11 +28,12 @@ const dexMetadata: HashConnectTypes.AppMetadata = {
 */
 const OPERATOR_ACCOUNT_ID = "0.0.47549759";
 const OPERATOR_PUBLIC_KEY = "302a300506032b6570032100a1dcb686465f9f4c22baaff9e321f057e82b5081e1d110969e7d512157979ab8";
-const OPERATOR_PRIVATE_KEY =
-  "302e020100300506032b657004220420ade33088ec47cd9d4ee4d9c9409567a4e39fddc2e4434aa6ace16e3278e253d5";
+const OPERATOR_PRIVATE_KEY = Uint8Array.from(
+  Buffer.from("302e020100300506032b657004220420ade33088ec47cd9d4ee4d9c9409567a4e39fddc2e4434aa6ace16e3278e253d5", "hex")
+);
 
 const operatorId = AccountId.fromString(OPERATOR_ACCOUNT_ID);
-const operatorKey = PrivateKey.fromString(OPERATOR_PRIVATE_KEY);
```

Comment:
I was getting a **BadKeyError: invalid private key length: 48 bytes** error when using `PrivateKey.fromString` since  OPERATOR_PRIVATE_KEY is in a hex format. Transforming the hex string into an uint and parsing from bytes fixed the issue.